apis/core/v1alpha1/condition.go

```diff
@@ -262,7 +261,7 @@ func ReconcileError(err error) Condition {
 }
 
 // ReferenceResolutionSuccess returns a condition indicating that Crossplane
-// successfully resolved the references used in the managed resource.
+// successfully resolved the references used in the resource.
```

Comment:
Do we use CRR in app stacks? I guess it makes sense to make this not specific to managed resources even if we don't.

Reply:
Yes, we will use it here https://github.com/crossplaneio/sample-stack-wordpress/pull/20

But I didn't open this PR just for that, I think `condition`s have become our way of communicating the state now (similar to k8s) and wording needs to be adjusted for that.